gamemodes/terrortown/gamemode/shared/lang/german.lua

```diff
@@ -1240,13 +1061,169 @@ L.door_locked = "Diese Tür ist verschlossen"
+L.label_crosshair_ironsight_low_enabled = "Senke Waffe beim Zielen durch Kimme und Korn"
+L.label_damage_indicator_enable = "Aktiviere Schadensanzeige"
+L.label_damage_indicator_mode = "Schadensanzeigen Thema"
+L.label_damage_indicator_duration = "Sekunden, die die Schadensanzeige sichtbar ist"
+L.label_damage_indicator_maxdamage = "Erlittener Schaden für maximale Sichtbarkeit"
+L.label_damage_indicator_maxalpha = "Maximaler Transparenzwert"
+L.label_performance_halo_enable = "Zeichne eine Linie um einige Entities, die man anschaut"
+L.label_performance_spec_outline_enable = "Aktiviere Zuschauermrandungen"
+L.label_performance_ohicon_enable = "Aktiviere überkopf Rollenicons"
+L.label_interface_tips_enable = "Zeige Tipps zum Spiel während des Zuschauens am unteren Bildschirmrand"
+L.label_interface_popup = "Dauer des Popups mit Infos am Anfang einer Runde"
+L.label_interface_fastsw_menu = "Aktiviert das Waffenwechselmenü, selbst wenn der schnelle Waffenwechsel aktiv ist"
+L.label_inferface_wswitch_hide_enable = "Aktiviere das automatische Schließen des Waffenwechselmenüs"
+L.label_inferface_scues_enable = "Spiele einen Ton ab, wenn eine Runde beginnt oder endet"
+L.label_gameplay_specmode = "Nur-Zuschauer-Modus (bleibe immer Zuschauer)"
+L.label_gameplay_fastsw = "Schneller Waffenwechsel"
+L.label_gameplay_hold_aim = "Aktiviere Halten zum Anvisieren"
+L.label_gameplay_mute = "Stelle lebende Spieler stumm, wenn du tot bist"
+L.label_gameplay_dtsprint_enable = "Deaktiviere Double-Tap Sprinten"
+L.label_gameplay_dtsprint_anykey = "Setze Double-Tap Sprinten fort, bis du stehen bleibst"
+L.label_hud_default = "Standard HUD"
+L.label_hud_force = "Erzwungenes HUD"
+
+L.label_bind_weaponswitch = "Waffe aufheben"
+L.label_bind_sprint = "Sprinten"
+L.label_bind_voice = "Globaler Sprachchat"
+L.label_bind_voice_team = "Team Sprachchat"
+
+L.label_hud_basecolor = "Grundfarbe"
+
+L.label_menu_not_populated = "Dieses Untermenü enthält keinen Inhalt."
+
+L.header_bindings_ttt2 = "TTT2 Tastenbelegungen"
+L.header_bindings_other = "Andere Tastenbelegungen"
+L.header_language = "Spracheinstellungen"
+L.header_global_color = "Wähle globale Farbe"
+L.header_hud_select = "Wähle ein HUD"
+L.header_hud_customize = "Passe das HUD an"
+L.header_vskin_select = "Wähle und passe den VSkin an"
+L.header_targetid = "TargetID Einstellungen"
+L.header_shop_settings = "Ausrüstungs Shop Einstellungen"
+L.header_shop_layout = "Layout der Elementliste"
+L.header_shop_marker = "Element Symbol Einstellungen"
+L.header_crosshair_settings = "Fadenkreuz Einstellungen"
+L.header_damage_indicator = "Schadensanzeigen Einstellungen"
+L.header_performance_settings = "Performance Einstellungen"
+L.header_interface_settings = "Anzeigeeinstellungen"
+L.header_gameplay_settings = "Spieleinstellungen"
+L.header_roleselection = "Aktierung Rollenzuweisung"
+L.header_hud_administration = "Wähle standard und erzwungene HUDs"
```

Comment:
```suggestion
L.header_hud_administration = "Wähle Standard-HUD und erzwungene HUDs"
```

Reply:
No, I don't like your suggestion. Either it should be my version or `"Wähle Standard-HUD und Erzwungenes-HUD"`

Reply:
Then I'd take your last suggestion